docs/release-notes/release-notes-0.17.0.md

```diff
@@ -218,6 +218,8 @@
 
 * [Stop rejecting payments that overpay or over-timelock the final
   hop](https://github.com/lightningnetwork/lnd/pull/7768).
+  
+* [Stop rejecting payments that overpay or over-timelock the final hop](https://github.com/lightningnetwork/lnd/pull/7768)
```

Comment:
Looks like a rebase issue, this is now duplicated.

Reply:
Fixed, sry.